projects/full-stack-checklist.md

```diff
@@ -0,0 +1,111 @@
+# PERN App Creation Checklist
+
+### Introduction
+
+This checklist covers how to create a full-stack app with a React frontend, Node/Express backend, and Postgres database (PERN) from scratch.
+
+### Prerequisites
+
+- Node
+- [Docker Desktop](https://docs.docker.com/get-docker/), if you want to use Docker for your Postgres database. If you are planning to deploy this application or collaborate on the code with other users, a Docker-ized database will be easier to maintain.
```

Comment:
@alodahl - Should this checklist include any Docker? The current cohort is using PERN with Docker to deploy to Heroku.

Reply:
@alodahl @vegetabill I think we can probably let this checklist go for now.  Cristina and I are working on updating how we will do the final project and eventonica (aka what we want to use as the base project for it). I think it will either land on George's set up or something similar but simplified. So when we get that worked out we can update the checklist to match the steps they will need for that.

Reply:
shall we merge what’s here and update when the time comes @SarahCawley ?

Reply:
@alodahl Yep that sounds like a good idea.